### PR TITLE
Remove PPU/SPU LLVM options 

### PR DESCRIPTION
**Pull request by raven02**
Since LLVM has been removed in 0.0.0.9 , to get less confusion on some users claimed the LLVM option got grey out .We can always re-enable it easily when LLVM implemented .

### CODE REVIEW

**Comment by elfogames** [comment]
something like this on rpcs3 would help too

https://github.com/citra-emu/citra/pull/1874

File: rpcs3/rpcs3.cpp [hunk 1]
```diff
@@ -96,7 +96,7 @@ cfg::map_entry<std::function<std::shared_ptr<GSRender>()>> g_cfg_gs_render(cfg::
 	{ "Null", PURE_EXPR(std::make_shared<NullGSRender>()) },
 	{ "OpenGL", PURE_EXPR(std::make_shared<GLGSRender>()) },
 #ifdef _MSC_VER
-	{ "DX12", PURE_EXPR(std::make_shared<D3D12GSRender>()) },
+	{ "DirectX 12", PURE_EXPR(std::make_shared<D3D12GSRender>()) },
```
Comment (11DH11) [inline rpcs3/rpcs3.cpp]: We do not use all DirectX in renderer, just Direct3D part. So that name incorrect too

Comment (raven02) [inline rpcs3/rpcs3.cpp]: Fixed as proposed

**Comment by Nekotekina** [comment]
@raven02 Does it really matter? Someone may ask "why LLVM completely disappeared" in the same way.

**Comment by raven02** [comment]
Yes 0.0.0.9 should be already stated we removed the LLVM completely

**Comment by raven02** [comment]
closing it . looks like LLVM PPU is coming

**Pull request closed without merging**